.gitignore

```diff
@@ -3,6 +3,8 @@
 .vscode/
 *.iml
 *.sublime*
+*.css
```

Comment:
just be aware of case when you can ignore needed .css files(you may not face it if all your files is scss)